src/codemagic/tools/_app_store_connect/action_groups/review_submissions_actions_group.py

```diff
@@ -95,3 +127,33 @@ def confirm_review_submission(
             should_print,
             submitted=True,
         )
+
+    @cli.action(
+        'list',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        AppStoreVersionArgument.PLATFORM,
+        ReviewSubmissionArgument.REVIEW_SUBMISSION_STATE,
+        action_group=AppStoreConnectActionGroup.REVIEW_SUBMISSIONS,
+    )
+    def list_review_submissions(
+        self,
+        application_id: ResourceId,
+        platform: Optional[Platform] = None,
+        review_submission_state: Optional[Union[ReviewSubmissionState, Sequence[ReviewSubmissionState]]] = None,
+        should_print: bool = True,
+    ) -> List[ReviewSubmission]:
+        """
+        Find and list review submissions in App Store Connect
```

Comment:
Again, I think it would be better if the docstring mentioned that it lists submission for specific app only.